decidim-admin/app/assets/stylesheets/decidim/admin/extra/_cards.scss

```diff
@@ -11,3 +11,16 @@ form{
     overflow: initial;
   }
 }
+
+// Newsletters index
+.text-right{
```

Comment:
if the css target is only the newsletter form, I'd rename `.text-right` to a more specific class so you assure it will only affect the newsletter form. ex: `.form.newsletter_deliver`